storybook/src/components/Radio/Radio.stories.tsx

```diff
@@ -64,33 +64,92 @@ export const RadioGroup: Story = {
     disabled: {
       table: { disable: true },
     },
-    invalid: {
-      table: { disable: true },
-    },
     onChange: {
       action: 'clicked',
       table: { disable: true },
     },
   },
-  render: () => (
-    <FieldSet legend="Waar gaat uw melding over?">
-      <Radio name="soort" value="horecabedrijf">
-        Horecabedrijf
-      </Radio>
-      <Radio name="soort" value="ander_soort_bedrijf">
-        Ander soort bedrijf
-      </Radio>
-      <Radio name="soort" value="evenement">
-        Evenement
-      </Radio>
-      <Radio name="soort" value="iets_anders">
-        Iets anders
-      </Radio>
+  render: (args) => (
+    <FieldSet
+      aria-describedby={`description1${args.invalid ? ' error1' : ''}`}
+      invalid={args.invalid}
+      legend="Waar gaat uw melding over?"
+      role="radiogroup"
+    >
+      <Paragraph className="ams-mb--sm" id="description1" size="small">
+        De laatstgenoemde melding.
+      </Paragraph>
+      {args.invalid && (
+        <ErrorMessage className="ams-mb--sm" id="error1">
+          Geef aan waar uw laatstgenoemde melding over gaat.
+        </ErrorMessage>
+      )}
+      <Column gap="extra-small">
+        <Radio invalid={args.invalid} name="about" value="horeca">
+          Horecabedrijf
+        </Radio>
+        <Radio invalid={args.invalid} name="about" value="ander_bedrijf">
+          Ander soort bedrijf
+        </Radio>
+        <Radio invalid={args.invalid} name="about" value="evenement">
+          Evenement
+        </Radio>
+        <Radio invalid={args.invalid} name="about" value="anders">
+          Iets anders
+        </Radio>
+      </Column>
     </FieldSet>
   ),
-  parameters: {
-    docs: {
-      source: { type: 'dynamic' },
+}
+
+export const InAnInvalidFieldSet: Story = {
+  args: {
+    invalid: true,
+  },
+  argTypes: {
+    checked: {
+      table: { disable: true },
+    },
+    children: {
+      table: { disable: true },
+    },
+    disabled: {
+      table: { disable: true },
+    },
+    onChange: {
+      action: 'clicked',
+      table: { disable: true },
     },
   },
+  render: (args) => (
+    <FieldSet
+      aria-describedby={`description2${args.invalid ? ' error2' : ''}`}
```

Comment:
This isn’t the improvement I was hoping for, is it?

```suggestion
      aria-describedby={['description2', args.invalid ? 'error2' : undefined].join('')}`}
```

Reply:
It's not much clearer imo, and the current code doesn't work :) You could join with `' '`, but that means you get this when `invalid` is false: `"description2 "`